scripts/install/install_macos.sh

```diff
@@ -542,6 +545,11 @@ uninstall()
   info "Untapping formula..."
   brew untap observiq/homebrew-observiq-otel-collector > /dev/null 2>&1 || error_exit "$LINENO" "Failed to untap formula"
   succeeded
+
+  info "Removing config files..."
+  rm -f $BREW_ETC/observiq_config.yaml.default > /dev/null 2>&1
```

Comment:
`BREW_ETC` can be replaced with `$(brew --prefix)/etc/`. There are other examples in here using the `brew --prefix`

Reply:
@cpheps fixed